README.md

```diff
@@ -85,7 +85,13 @@ module.exports = {
 			    test: /\.svg/,
 			    use: {
 			        loader: 'svg-url-loader',
-			        options: {}
+			        options: {
+				        // => Optional supported parameters, see above
+				        stripdeclarations: true,
+				        encoding: 'base64',
+				        iesafe: true
+				        // ...
+			        }
```

Comment:
I don't want to include parameters here, because people may think that they have to specify all of them to make loader work.  
In most cases you should be able to use loader without any options (if that is not the case, that will mean that we have bad defaults, which should be fixed).

Instead I'd like to have a set of examples describing some common scenarios.  
For example:
> Scenario 1
> Alice has some SVG icons and want to use them as background images in her css files. She uses webpack and doesn't care about IE.
> _... and then comes webpack config for this specific use case_
> 
> Scenario 2
> Alice uses SVG icons but with standard webpack config her icons does not appear under IE
> _... and then comes webpack config with `iesafe: true`_

If you have time to put together something like this, that will be awesome.